### PR TITLE
fix: Update github->actions workflow

### PR DESCRIPTION
**Pull request by tdrozdovsky**
# Description

Fix security  issues: "Token-Permissions"

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code cleanup/refactoring
- [ ] Documentation update
- [ ] This change requires a documentation update
- [x] CI system update
- [ ] Test Coverage update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/publish-mvn-pkg.yml [hunk 1]
```diff
@@ -1,8 +1,12 @@
-name: Publish package to GitHub Packages
+name: Publish Maven package to GitHub Packages
```
Comment (m-rudyk) [inline .github/workflows/publish-mvn-pkg.yml:1]: It is better to leave as before, since it is not maven package.

**Review by m-rudyk — APPROVED** [review]
Approved. Please check single comment.

**Review by o-kopysov — APPROVED** [review]
LGTM

**Review by o-konoval — APPROVED** [review]
+1

**Pull request merged**